docs/includes/giza.log

```diff
@@ -0,0 +1,21 @@
+WARNING:giza.content.changelog.tasks:changelog generation is not configured.
```

Comment:
AFAIK, `.DS_Store` is OSX-specific and `*~` is specific to some editors. I'll likely remove those, as they are generally best left to developers' own `.gitignore` files (in their home directories). The other paths look like good additions, though. Thanks!